### PR TITLE
fix: Small readability improvement on "Installation.md"

### PR DESCRIPTION
**Pull request by BarbzYHOOL**
Just check the changed file https://github.com/azerothcore/wiki/blob/717bce416432348a4fd1147072cd8c9fad274af6/docs/Installation.md

### CODE REVIEW

**Comment by BarbzYHOOL** [comment]
Finally I put all links to installation in one page, we have too many useless links, it's scattered everywhere it's bad

**Comment by Helias** [comment]
- I don't like much the UPPERCASE titles.
- I suggest to leave " 1) Installing the requirements" instead of "Requirements".
- Only the debug build of MacOS is broken, the release one should works.

For the UPPECASE titles, at least, I think that we can use  header2 (##) or header3 (###) instead of header1 (#)

**Comment by BarbzYHOOL** [comment]
1) i don't like the lowercase titles, they are unnoticeable, unremarkable
who's right then ?

More # = smaller font size, so no.

Imo, it looks good here https://github.com/azerothcore/wiki/blob/717bce416432348a4fd1147072cd8c9fad274af6/docs/Installation.md these are the big titles and they are easily seen since they are no longer in the sidebar, they must be easily found (hence why i put them at the top too, instead of at the bottom but at the bottom could be good too)

2) of course we "install the requirements" we are on the "installation page"

why add extra words that repeat the same thing? it's for developers, people with logic and structured mind, extra useless words is a waste of time and it's a source of confusion, it's not good for visual memory either (some of our tutorials are way too wordy and too lengthy and each time we need to read everything just to find the line we want)


3) no idea for MacOS, i just copied what was written

**Comment by Helias** [comment]
:ok:, for me it's the same :P

**Comment by Stoabrogga** [comment]
I also don't like UPPERCASE titles, could you please change this? It's like the documentation is screaming at me. It already suffices to use H1 headers which are already too big in my opinion.

**Comment by BarbzYHOOL** [comment]
> I also don't like UPPERCASE titles, could you please change this? It's like the documentation is screaming at me. It already suffices to use H1 headers which are already too big in my opinion.

poor fragile creature

no for real, when I put them normal, it's very hard to distinguish the hierarchy between titles, that's why I do that, and so I disagree so I don't want to change it back lol

**Comment by Stoabrogga** [comment]
Do you have an example of how this would look like? What I see in the preview on Github is just plain ugly, sorry.

**Comment by heyaapl** [comment]
This is still pending merge @FrancescoBorzi Any chance this can get merged in so Barbz's changes would be great on the install page.

**Comment by FrancescoBorzi** [comment]
I'd also prefer not using uppercase, however, I'm not a UI expert and I don't want this PR to hang for this trivial detail. Merged.

image

**Comment by BarbzYHOOL** [comment]
okay, maybe make an issue "change uppercase to lowercase" and add "vote with :+1:  or :-1: " xD
